README.md

```diff
@@ -920,6 +938,251 @@ branch. Right? **Right?**
+or need to make changes to your branch. If, for example, Chris realizes here
+that the tests are failing because they have the alphabetical order wrong,
+they'll need to pull down these changes, fix the alphabetical order, push back
+up, and attempt to merge again.
+
+##### You need a review
+
+Now you've finally gotten your code up-to-date and all the checks pass. You
+still can't merge because of a big red X and the message "Review required".
+This is because we're following a common "best practice" and requiring at
+least one code review _by another person_ for each pull request.
+
+So you need to get someone to review your code! In the upper right there's
+a "Reviewers" section. Click on the gear there and you should get a drop down
+with the (GitHub) names of the other people in the class. Choose two or three
+and click their names to invite them to review your code. _Don't just choose
+the top three – if everyone does that the same few people will be asked to do
+_all_ the reviews. Instead, try choosing the two or three names right after yours
+on the list – that should spread the load around.
+
+At this point, you can't really do anything until folks have approved your
+pull request, so now's a good time to review some other people's pull requests!
+
+##### Review some other pull requests
+
+While you're waiting for feedback from other people, let's do some code reviews
+on other people's work. You might have been invited to provide some code reviews;
+check your notifications (the bell in the far top right) to see if there are any
+code review invitations there. Otherwise you can click on the project's "Pull
+requests" tab at the top, and pick any option pull request.
+
+If you had been invited to review this pull request, then there should be a note
+to that effect near the top, and a green "Add your review" button which you can
+click to start your review. If you haven't been invited, you can just click on the
+"Files changed" tab of the pull request, and that will take you to the same place.
+
+You'll now be looking at all the changes (old on the left, new on the right).
+There's a _lot_ you can do here (read up on this), but for the moment we'll just
+look over the code and make sure the changes look reasonable. Some things to
+check for:
+
+* [ ] Does it look (by eye) that this will print all the greetings in alphabetical
+  order? (The tests check that for sure, but you should always be thinking about
+  correctness as well.)
+* [ ] Do any new methods they're adding have reasonable names that convey useful
+  info to the reader?
+* [ ] Are new method names in camel cases, starting in lowercase (the Java standard
+  for method names)?.
+* [ ] Is the implementation of the any new methods reasonable?
+* [ ] Is the spacing and indentation uniform and consistent with the rest of the file?
+
+Etc., etc.
+
+When you've looked over the code, click the green "Review changes" button on the
```

Comment:
Do we want to mention that you can comment on individual lines (and groups of lines)?

Reply:
I thought about this, but quickly one descends into documenting all the _many_ features of the code review system, and that seems to both duplicate materials that GitHub has written and a road to madness. I might demo a code review in lab, and show them some of those nifty tools.